tests/acceptance/registration-error-test.js

```diff
@@ -0,0 +1,62 @@
+import { module, test } from 'qunit';
+import { find, visit } from '@ember/test-helpers';
+import { t } from 'ember-intl/test-support';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import Service from '@ember/service';
+import { setupRequiredEndpoints } from '../helpers/acceptance-utils';
+
+class IntegrationStub extends Service {
+  async configure(user) {
+    this.currentUser = user;
+  }
+
+  isPendoEnabled() {
+    return false;
+  }
+
+  isCrispEnabled() {
+    return false;
+  }
+}
+
+module('Acceptance | registration error', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    await setupRequiredEndpoints(this.server);
+
+    this.owner.register('service:integration', IntegrationStub);
+  });
+
+  test('it redirects via-partner-invite to error page when token is invalid', async function (assert) {
+    await visit('/register-via-invite/invalid');
+
+    assert.dom('[data-test-token-error]').exists();
+
+    assert.dom('[data-test-error-SvgIcon]').exists();
+
+    assert.dom('[data-test-error-text]').hasText(t('somethingWentWrong'));
+
+    assert.strictEqual(
+      find('[data-test-error-helperText]').innerHTML,
+      t('invalidTokenError', { htmlSafe: true }).toString()
+    );
+  });
+
+  test('it redirects via-org-invite to error page when token is invalid', async function (assert) {
+    await visit('/invite/invalid');
```

Comment:
instead of invalid pass a token using `faker` and mock the api response for invalid token